src/lib/comms/include/sol-http-server.h

```diff
@@ -68,6 +68,16 @@ struct sol_http_server;
 struct sol_http_request;
 
 /**
+ * @struct sol_http_response_sse
+ * @brief Opaque handler for response using server-sent-event.
+ *
+ * This response is create with @c sol_http_server_send_response_sse
+ * and data can be given using @c sol_http_server_reponse_sse_send_data,
+ * to delete it use @c sol_http_server_reponse_ss2_del.
+ */
+struct sol_http_response_sse;
```

Comment:
In Golang, there's the `Flusher` interface that the `http.ResponseWriter` object might implement. This interface has only one method, `Flush()`, which sends the currently-buffered response to the socket. I like this name, it's similar to other streaming APIs, such as `fflush()`.